### PR TITLE
VR > Customizer > Add accent color & move link color 

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://moderntribe.atlassian.net/browse/TEC-3067
🎫 https://moderntribe.atlassian.net/browse/TEC-3069
🎥 https://www.loom.com/share/03fb8a858bed44658ac4131ed51d5d9b

### CODE REVIEW

**Comment by juanfra** [comment]
@bordoni can I please get a CR on this one? Thanks!

File: src/Tribe/Customizer/Global_Elements.php [hunk 1]
```diff
@@ -31,12 +31,195 @@ public static function instance() {
...
+			// Pro styles.
+			$template .= '
+				.tribe-events-pro .tribe-events-pro-week-grid__header-column--current .tribe-events-pro-week-grid__header-column-daynum,
+				.tribe-events-pro .tribe-events-pro-week-grid__header-column--current .tribe-events-pro-week-grid__header-column-daynum-link {
+					color: <%= global_elements.accent_color %>;
+				}
+			';
+
+			$template .= '
+				.tribe-events-pro.tribe-events-view--week .datepicker .day.current::before {
```
Comment (bordoni) [inline src/Tribe/Customizer/Global_Elements.php]: Pro Style should be added from Pro.

Comment (bordoni) [inline src/Tribe/Customizer/Global_Elements.php]: Pro Style should be added from Pro.

File: src/Tribe/Customizer/Global_Elements.php [hunk 2]
```diff
@@ -31,12 +31,195 @@ public static function instance() {
...
+			$template .= '
+				.tribe-events-pro .tribe-events-pro-week-grid__multiday-event-bar-inner,
+				.tribe-events .tribe-events-calendar-month__multiday-event-bar-inner {
+					background-color: rgba( ' . $accent_css_rgb . ', 0.24);
+				}
+			';
+
+			$template .= '
+				.tribe-events .tribe-events-calendar-month__multiday-event-bar-inner--hover,
+				.tribe-events-pro .tribe-events-pro-week-grid__multiday-event-bar-inner--hover,
```
Comment (bordoni) [inline src/Tribe/Customizer/Global_Elements.php]: Pro Style should be added from Pro.

**Comment by juanfra** [comment]
@bordoni PRO styles were moved to PRO https://github.com/moderntribe/events-pro/pull/1483 - Can I please have another CR here? Thanks 🎉

**Pull request merged**